lib/aruba/environments/unix_environment.rb

```diff
@@ -0,0 +1,11 @@
+require 'aruba/environments/basic_environment'
+
+module Aruba
+  module Environments
+    class UnixEnvironment < BasicEnvironment
+      def match?(*)
+        Aruba::Platform.unix?
```

Comment:
My preference would just be to call `platform.environment_variables`. We only need to make the windows / unix decision once, and everything should fall out from that.